extract/extract.go

```diff
@@ -271,7 +272,7 @@ func Endorsement(opts *Options) (out []byte, err error) {
 		if quoteErr != nil {
 			return nil, quoteErr
 		}
-		if len(endorsement) > 0 {
+		if len(endorsement) > 0 && !opts.ForceFetch {
```

Comment:
Why is the ForceFetch check in here instead of being on line 265 ("if opts.Provider != nil && objectName == """)?

Reply:
It allows the object name to be derived from the measurement in the quote.